packaging/resource_suite_s3_nocache.py

```diff
@@ -2198,6 +2198,73 @@ def test_iput_and_icp_with_invalid_secret_key__issue_6154(self):
                     'DATA_REPL_STATUS', '0'])
             self.user0.run_icommand(['irm', '-f', logical_path])
 
+    def test_s3_in_replication_node__issue_2102(self):
```

Comment:
Consider using more words so that the intent of the test is not ambiguous.

Reply:
done